autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/pixel/AutofillPixelNames.kt

```diff
@@ -168,6 +180,19 @@ enum class AutofillPixelNames(override val pixelName: String) : Pixel.PixelName
     AUTOFILL_SITE_BREAKAGE_REPORT_CONFIRMATION_DISPLAYED("autofill_logins_report_confirmation_displayed"),
     AUTOFILL_SITE_BREAKAGE_REPORT_CONFIRMATION_DISMISSED("autofill_logins_report_confirmation_dismissed"),
     AUTOFILL_SITE_BREAKAGE_REPORT_CONFIRMATION_CONFIRMED("autofill_logins_report_confirmation_confirmed"),
+
+    AUTOFILL_SERVICE_ENABLED("autofill_extension_enabled"),
+    AUTOFILL_SERVICE_ENABLED_DAU("autofill_extension_toggled_on"),
+    AUTOFILL_SERVICE_DISABLED("autofill_extension_disabled"),
+    AUTOFILL_SERVICE_DISABLED_DAU("autofill_extension_toggled_off"),
+    AUTOFILL_SERVICE_SUGGESTION_CONFIRMED("autofill_extension_quicktype_confirmed"),
+    AUTOFILL_SERVICE_PASSWORDS_OPEN("autofill_extension_passwords_opened"),
+    AUTOFILL_SERVICE_PASSWORDS_DISMISSED("autofill_extension_passwords_dismissed"),
+    AUTOFILL_SERVICE_PASSWORDS_DISMISSED_AUTH("autofill_extension_passwords_dismissed_auth"),
+    AUTOFILL_SERVICE_PASSWORD_SELECTED("autofill_extension_password_selected"),
+    AUTOFILL_SERVICE_PASSWORDS_SEARCH("autofill_extension_passwords_search"), // doesn't apply, we automatically focus the search bar
```

Comment:
the comment is confusing or incomplete; “doesn’t apply” lacks some context about what doesn’t apply

I think it’s indicating this isn’t fired _automatically_ and requires user-action based on https://app.asana.com/0/0/1209115687167237/1209505022666633/f, and the comment could be clearer

Reply:
Agree, I initially considered not triggering this at all, and left the comment. But then changed my mind. I will remove the comment.